client/components/SimilarListings.jsx

```diff
@@ -33,12 +34,33 @@ class SimilarListings extends React.Component {
     }
   }
 
+  nextSlide() {
+    this.setState({
+      listings: this.state.listings,
+      position: this.state.position + 1,
+      transform: -100*(this.state.position+1),
+    });
+  }
+
+  prevSlide() {
+    this.setState({
+      listings: this.state.listings,
+      position: this.state.position -1,
+      transform: -100*(this.state.position-1),
+    });
+  }
+
   render() {
     return (
       <div className="similar">
-        {this.state.listings.map(listing => (
-          <Listing listing={listing} />
-        ))}
+        <h1>Similar listings</h1>
```

Comment:
Just pointing out semantics: Your index.html is titled "More like this", but you have "Similar listings" as the h1 here.

Reply:
You're right, I will go change the index.html in my next branch.